_posts/2015-11-03-november-newsletter.md

```diff
@@ -0,0 +1,296 @@
+interface. He decided to write a new plugin from the scratch because Neovim
+doesn't support the lua interface required for neocomplete. @Shougo is the sith
+lord of Vim plugins, here's a list containing some of his previous work:
+
+- [neocomplete][neocomplete]
+- [neocomplcache][neocomplcache]
+- [vimshell][vimshell]
+- [unite][unite]
+- [neobundle][neobundle]
+
+The community can expect great things from [deoplete.nvim][deoplete]!
+
+### Neoterm
+
+[Neoterm][neoterm] is a plugin for easily running tests in a [terminal
+window][terminal-emulator]. It was written by @kassio and supports the following
+test libraries:
+
+- rspec
+- cucumber
+- minitest
+- go-lang test
+- nose
+
+Very useful @kassio!
+
+[terminal-emulator]: https://neovim.io/doc/user/nvim_terminal_emulator.html#nvim-terminal-emulator
+[0.1-milestone]: https://github.com/neovim/neovim/milestones/0.1-first-public-release
+[history-of-salt]: https://en.wikipedia.org/wiki/History_of_salt
+[first-campaign]: https://www.bountysource.com/teams/neovim
+[salt-campaign]: https://salt.bountysource.com/teams/neovim
+[shada-proposal]: https://github.com/neovim/neovim/issues/999
+[shada-pr]: https://github.com/neovim/neovim/pull/2506
+[event-loop-pr]: https://github.com/neovim/neovim/pull/2980
+[jemalloc]: http://www.canonware.com/jemalloc/
+[jemalloc-4-pr]: https://github.com/neovim/neovim/pull/3289
+[xdg-spec]: http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
+[xdg-pr1]: https://github.com/neovim/neovim/pull/3120
+[xdg-pr2]: https://github.com/neovim/neovim/pull/3198
+[xdg-pr3]: https://github.com/neovim/neovim/pull/3470
+[xdg-proposal]: https://github.com/neovim/neovim/issues/78
+[travis-container]: http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+[container-pr]: https://github.com/neovim/neovim/pull/2938
+[quickbuild]: http://freecode.com/projects/quickbuild
+[syntastic]: https://github.com/scrooloose/syntastic
+[neomake]: https://github.com/benekastah/neomake
+[job-control]: https://neovim.io/doc/user/job_control.html#job-control
+[deoplete]: https://github.com/Shougo/deoplete.nvim
+[neoterm]: https://github.com/kassio/neoterm
+[remote-plugin]: https://neovim.io/doc/user/remote_plugin.html#remote-plugin
+[vimshell]: https://github.com/Shougo/vimshell.vim
+[unite]: https://github.com/Shougo/Unite.vim
+[neocomplete]: https://github.com/Shougo/neocomplete.vim
+[neocomplcache]: https://github.com/Shougo/neocomplcache.vim
+[neobundle]: https://github.com/Shougo/neobundle.vim
+[fzf]: https://github.com/junegunn/fzf
+[fzf.vim]: https://github.com/junegunn/fzf.vim
+[vim-plug]: https://github.com/junegunn/vim-plug
+[ctrlp]: https://github.com/kien/ctrlp.vim
+[windows-instrutions]: https://github.com/neovim/neovim/wiki/Installing-Neovim#windows
+[neovim-qt]: https://github.com/equalsraf/neovim-qt
+[homebrew-formula]: https://github.com/neovim/homebrew-neovim
+[archlinux-pkgbuild]: https://aur.archlinux.org/packages/neovim
```

Comment:
Neovim has been added to the Arch Linux repos, so this link can be changed to https://www.archlinux.org/packages/?q=neovim